_data/tools.yml

```diff
@@ -1872,3 +1872,16 @@
   v2: true
   v3: true
   v3_1: true
+
+- name: Karate-IDE
+  category: 
+    - converters
+    - testing
+    - mock
+  link: https://marketplace.visualstudio.com/items?itemName=KarateIDE.karate-ide
+  github: https://github.com/ivangsa/karate-ide
+  language: VSCode Extension
+  description:
+    Generate [Karate](https://github.com/karatelabs/karate) Tests and Mocks from OpenAPI 3.0 documents and so you can quickly test/explore your API.
```

Comment:
this isn't a markdown based project, so this will be formatted badly.

Reply:
> this isn't a markdown based project, so this will be formatted badly. 

I see... I think I copied from `oapi-codegen` that uses a link in the description.. 

I'll remove now the link from de description to match all other tools